pkg/bindings/down/play.go

```diff
@@ -0,0 +1,39 @@
+package down
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	"github.com/containers/podman/v4/pkg/bindings"
+	"github.com/containers/podman/v4/pkg/domain/entities"
+	"github.com/sirupsen/logrus"
+)
+
+func KubeDown(ctx context.Context, path string) (*entities.PlayKubeReport, error) {
+	// TODO: should this be entities.PlayKubeTeardown?
+	var report entities.PlayKubeReport
+	conn, err := bindings.GetClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := f.Close(); err != nil {
+			logrus.Warn(err)
+		}
+	}()
+	response, err := conn.DoRequest(ctx, f, http.MethodDelete, "/play/kube", nil, nil)
```

Comment:
This is the old action, a delete of the play, which still seems like a reasonable way to implement this handler (POST on play is creating the play, DELETE on play is tearing down the play)